packages/back-end/types/organization.d.ts

```diff
@@ -147,6 +147,7 @@ export interface OrganizationSettings {
   pValueThreshold?: number;
   /** @deprecated */
   implementationTypes?: ImplementationType[];
```

Comment:
Remove this since an org level flag isn’t being used.

Reply:
oops, left over from earlier commit. removed